### ISSUE TITLE
Timestamp Error scheduleByTimestampNt too far

### ISSUE THREAD

**Issue by BeerMoneyMotorsports** [rusefi/rusefi#3395]
I have experienced this along with Hellen users.  Can we fix this annoyance?  Complete power cycle resolves the issue.  

1-started the engine
2-stopped the engine without powering down ECU
3-waited minutes
4-attempted to start engine again

![Image from iOS (2)](https://user-images.githubusercontent.com/82395131/138537266-014ddab1-a0bf-4840-9301-392890a67c66.jpg)

**Comment by rusefillc** [comment]
@BeerMoneyMotorsports please reproduce with latest snapshot, error message is more detailed now

**Comment by BeerMoneyMotorsports** [comment]
i was able to reproduce this recently.  i occurred when the laptop remained connected but closed.  I will attempt to reproduce again and get more detail.

**Comment by mck1117** [comment]
Happen to remember what it said? I think we added what system caused the error to the message.

**Comment by BeerMoneyMotorsports** [comment]
@mck1117

![image](https://user-images.githubusercontent.com/82395131/146274338-313de011-469b-4648-a640-f928790d5b4f.png)

**Comment by rusefillc** [comment]
@elDominio had it

![image](https://user-images.githubusercontent.com/48498823/159360555-98bbb965-b018-464e-b503-242f6eee8228.png)

**Comment by rusefillc** [comment]
@mck1117 can you have another look? at least time for more statements

**Comment by mck1117** [comment]
@NMSTEC had it
![image](https://user-images.githubusercontent.com/568254/168191145-044ec8af-00ee-4722-8a13-0974722bc865.png)

**Comment by mck1117** [comment]
all of these too far values seem to be close to 32b overflow, which hints at this maybe being an integer overflow somewhere.

log2(1274220776) = 30.2
log2(1395092012) = 30.4

**Comment by rusefillc** [comment]
very fresh master on proteus F4
I was playing with iDrive and got distracted unit was just idling connected to rusEFI console

```
2022-05-14_20_47_17_320: EngineState: CRITICAL error: scheduleByTimestampNt() too far: 3999995 scheduleForLater 20220514@proteus
2022-05-14_20_47_17_557: EngineState: CRITICAL error: scheduleByTimestampNt() too far: 3999995 scheduleForLater 20220514@proteus
2022-05-14_20_47_17_758: EngineState: CRITICAL error: scheduleByTimestampNt() too far: 3999995 scheduleForLater 20220514@proteus
```

![image](https://user-images.githubusercontent.com/48498823/168452761-33e68b6e-4689-453e-aa14-9fa555cbb0cb.png)

**Comment by rusefillc** [comment]
![image](https://user-images.githubusercontent.com/48498823/168454451-1f6a143d-976a-468d-a4e9-c89c18dc9970.png)

**Comment by rusefillc** [comment]
Fixed by #4174
